test.js

```diff
@@ -5,26 +5,34 @@ import isProgressive from './index.js';
 
 const getPath = name => `fixture/${name}.jpg`;
 
-test('.buffer()', t => {
+test.serial('.buffer()', t => {
```

Comment:
why serial?

Reply:
They're reading the same files as the other tests that are run in parallel. This blocks the reading of the files in the other tests making them slower or causes hangs where the file cannot be opened because it's being accessed asynchronously.

I benchmarked the methods by running each test multiple times and could only get it to behave consistently if the synchronous tests are run serially.

Reply:
👍